internal/loggers/console/console.go

```diff
@@ -0,0 +1,48 @@
+/*
+ * @license
+ * Copyright 2023 Dynatrace LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package console
+
+import (
+	"log"
+)
+
+var Instance = Logger{consoleLogger: log.Default()}
+
+// Logger is a wrapper around the go default logger that just logs to the console
```

Comment:
I think it would be a good place to mention that this one's generally unused in production, as zap is also used for plain/non-JSON logging.